test/e2e/elasticsearch_test.go

```diff
@@ -197,7 +204,7 @@ func getJaegerAllInOne(name string) *v1.Jaeger {
 
 func hasIndexWithPrefix(prefix string, esPort string) (bool, error) {
 	c := http.Client{}
-	req, err := http.NewRequest(http.MethodGet, "http://localhost:" + esPort + "/_cat/indices", nil)
+	req, err := http.NewRequest(http.MethodGet, "http://localhost:"+esPort+"/_cat/indices", nil)
```

Comment:
@jpkrohling we are maybe not running format on the test files

Reply:
Could you please create an issue for this, so that we don't lose track of the problem.

Reply:
done #595